libvast/builtins/formats/zeek_tsv.cpp

```diff
@@ -0,0 +1,757 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/arrow_table_slice.hpp"
+#include "vast/cast.hpp"
+#include "vast/concept/parseable/string/any.hpp"
+#include "vast/concept/parseable/vast/pipeline.hpp"
+#include "vast/concept/printable/to_string.hpp"
+#include "vast/concept/printable/vast/json.hpp"
+#include "vast/data.hpp"
+#include "vast/detail/string.hpp"
+#include "vast/detail/string_literal.hpp"
+#include "vast/detail/to_xsv_sep.hpp"
+#include "vast/detail/zeekify.hpp"
+#include "vast/generator.hpp"
+#include "vast/plugin.hpp"
+#include "vast/table_slice_builder.hpp"
+#include "vast/to_lines.hpp"
+#include "vast/type.hpp"
+#include "vast/view.hpp"
+
+#include <arrow/record_batch.h>
+#include <caf/error.hpp>
+#include <caf/expected.hpp>
+#include <caf/none.hpp>
+#include <fmt/core.h>
+#include <fmt/format.h>
+
+#include <algorithm>
+#include <cctype>
+#include <iterator>
+#include <string>
+#include <string_view>
+
+namespace vast::plugins::zeek_tsv {
+
+namespace {
+// The type name prefix to preprend to zeek log names when transleting them
```

Comment:
```suggestion
// The type name prefix to preprend to Zeek log names when translating them
```

Reply:
Fixed in https://github.com/tenzir/vast/pull/3114/commits/d01e63a216f6cf0cb80a82adb3a4ae4e89a9286c